Library/Homebrew/cmd/orphaned.rb

```diff
@@ -0,0 +1,13 @@
+#:  * `orphaned`:
+#:    Lists kegs that were installed as dependencies of other kegs, but are no
+#:    longer required for any explicitly-installed keg.
+
+require "keg"
+
+module Homebrew
+  module_function
+
+  def orphaned
+    puts Keg.orphaned.map { |k| "#{k.name} #{k.version}" }
```

Comment:
If version 1.0 of a formula is orphaned but 1.1 is required, that `brew orphaned | xargs brew uninstall` would remove 1.1 and leave 1.0.

With that in mind, would it be better to output the full keg path when not outputting to a TTY?